webrtc/src/benchmarkRtc.ts

```diff
@@ -1,14 +1,19 @@
 import { chromium, Browser } from "playwright";
+import { TrackEncoding } from "@jellyfish-dev/ts-client-sdk";
 import { Client } from "./client";
 import { Args } from "./types";
 import { getEncodingsReport, onEncodingsUpdate } from "./encodingReporter";
 
 const frontendAddress = "http://localhost:5005";
 const fakeVideo = "media/sample_video.mjpeg";
 const fakeAudio = "media/sample_audio.wav";
-const ENCODING_REPORT_PERIOD = 5;
-const INBOUD_TRACK_BANDWIDTH = 0.15 + 0.5 + 1.5;
-const OUTBOUND_TRACK_BANDWIDTH = 1.5;
+
+const encodingReportPeriod = 5;
+const encodingBitrate = new Map<TrackEncoding, number>([
+  ["l", 0.15],
+  ["m", 0.5],
+  ["h", 1.5],
+]);
```

Comment:
Why did you change the naming convention for these variables?

Reply:
Although there are many conventions for naming constants, I think I prefer the Camel Case, also other constants also follow this rule.